.github/workflows/chart-release.yaml

```diff
@@ -0,0 +1,34 @@
+name: Release Charts
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      - name: Configure Git
+        run: |
+          git config user.name "$GITHUB_ACTOR"
+          git config user.email "$GITHUB_ACTOR@users.noreply.github.com"
+      # See https://github.com/helm/chart-releaser-action/issues/6
+      - name: Install Helm
+        run: |
+          curl -fsSLo get_helm.sh https://raw.githubusercontent.com/helm/helm/master/scripts/get-helm-3
+          chmod 700 get_helm.sh
+          ./get_helm.sh
+      - name: Add dependency chart repos
+        run: |
+          helm repo add stable https://kubernetes-charts.storage.googleapis.com/
+          helm repo add incubator https://kubernetes-charts-incubator.storage.googleapis.com/
+      - name: Run chart-releaser
+        uses: helm/chart-releaser-action@v1.0.0
+        env:
+          CR_TOKEN: "${{ secrets.GITHUB_TOKEN }}"
```

Comment:
@olemarkus I'd suggest taking a look at the kubernetes-sigs/descheduler [release.yaml](https://github.com/kubernetes-sigs/descheduler/blob/master/.github/workflows/release.yaml) as it has the updated GitHub action versions and will allow you to use a custom tag for helm charts (with the `CR_RELEASE_NAME_TEMPLATE` env var) instead of the default (which would be `metrics-server-2.11.2` with the current config).